algorithms/appo/learner.py

```diff
@@ -109,6 +109,16 @@ def __init__(
 
         self.process = Process(target=self._run, daemon=True)
 
+        if is_continuous_action_space(self.action_space):
+            raise NotImplementedError(f'Continuous Action spaces not supported!')
+
+        if self.cfg.exploration_loss == 'entropy':
+            self.exploration_loss_func = self.entropy_exploration_loss
+        elif self.cfg.exploration_loss == 'symmetric_kl':
+            self.exploration_loss_func = self.symmetric_kl_exploration_loss
+        else:
+            raise NotImplementedError(f'{self.cfg.exploration_loss} not supported!')
+
```

Comment:
This is good

Reply:
We can actually also avoid checking for `self.cfg.exploration_loss_coeff` later and just check it here.
Just add another if in the beginning:

```
if self.cfg.exploration_loss_coeff == 0.0:
    self.exploration_loss_func = lambda action_distr: 0.0

```